features/zoek-met-burgerservicenummer/overzicht.feature

```diff
@@ -0,0 +1,223 @@
+    | burgerservicenummer     | 987654321                  |
+    | gemeenteVanInschrijving | 0800                       |
+    | fields                  | reisdocumentnummer         |
+    Dan heeft de response 0 reisdocumenten
+
+  Scenario: de persoon heeft meerdere reisdocumenten ingeleverd en heeft één reisdocument nog in bezit
+    Gegeven de persoon met burgerservicenummer '000000152' heeft een 'reisdocument' met de volgende gegevens
+    | naam                                                                    | waarde       |
+    | soort reisdocument (35.10)                                              | PN           |
+    | nummer reisdocument (35.20)                                             | NE3663258    |
+    | datum uitgifte Nederlands reisdocument (35.30)                          | 20131106     |
+    | datum einde geldigheid reisdocument (35.50)                             | 20231106     |
+    | datum inhouding dan wel vermissing Nederlands reisdocument (35.60)      | 20221229     |
+    | aanduiding inhouding dan wel vermissing Nederlands reisdocument (35.70) | I            |
+    En de persoon heeft een 'reisdocument' met de volgende gegevens
+    | naam                                           | waarde    |
+    | soort reisdocument (35.10)                     | PN        |
+    | nummer reisdocument (35.20)                    | NWE45TN71 |
+    | datum uitgifte Nederlands reisdocument (35.30) | 20230317  |
+    | datum einde geldigheid reisdocument (35.50)    | 20330317  |
+    En de persoon heeft een 'reisdocument' met de volgende gegevens
+    | naam                                                                    | waarde    |
+    | soort reisdocument (35.10)                                              | PN        |
+    | nummer reisdocument (35.20)                                             | PP82046A7 |
+    | datum einde geldigheid reisdocument (35.50)                             | 20281103  |
+    | datum inhouding dan wel vermissing Nederlands reisdocument (35.60)      | 20230205  |
+    | aanduiding inhouding dan wel vermissing Nederlands reisdocument (35.70) | I         |
+    En de persoon heeft de volgende 'verblijfplaats' gegevens
+    | gemeente van inschrijving (09.10) |
+    | 0800                              |
+    Als reisdocumenten wordt gezocht met de volgende parameters
+    | naam                    | waarde                     |
+    | type                    | ZoekMetBurgerservicenummer |
+    | burgerservicenummer     | 987654321                  |
+    | gemeenteVanInschrijving | 0800                       |
+    | fields                  | reisdocumentnummer         |
+    Dan heeft de response 1 reisdocumenten
+    En heeft de response een 'reisdocument' met de volgende gegevens
+    | naam               | waarde    |
+    | reisdocumentnummer | NWE45TN71 |
+
+  Scenario: de persoon heeft eerdere reisdocumenten ingeleverd en heeft verschillende soorten reisdocumenten in bezit
```

Comment:
Wat betekent 'eerdere' in de scenario? Is er een verschil tussen de vorige scenario waar 'eerdere' niet wordt genoemd in de scenario titel?

Reply:
eerdere kan er uit.